pkg/kubectl/stop_test.go

```diff
@@ -659,6 +661,24 @@ func TestSimpleStop(t *testing.T) {
 	}
 }
 
+func getPodTemplateSpecHash(template api.PodTemplateSpec) uint32 {
```

Comment:
it might just be cleaner to put this deploymentutil as `GetPodTemplateSpecHashV1(...)` which keeps this logic together.  It's not as if we're avoiding having to import deploymentutil.